cmd/cel-shed/eds_store_stress.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/libs/edssser"
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+)
+
+const (
+	edsStorePathFlag = "store-path"
+	edsWritesFlag    = "eds-writes"
+	edsSizeFlag      = "eds-size"
+	edsEnableLogFlag = "log"
+	edsStoreCleanup  = "cleanup"
+)
+
+func init() {
+	shareCmd.AddCommand(edsStoreStress)
+
+	edsStoreStress.Flags().String(edsStorePathFlag, "", "Directory path to use for stress test. os.TempDir by default.")
+	edsStoreStress.Flags().Int(edsWritesFlag, math.MaxInt, "Total EDS writes to make. MaxInt by default.")
```

Comment:
Given that we want to test store performance on certain amount of blocks stored, would it be easier to use this tool if this setting was rather "max amount of eds stored". So instead of local counter staring from 0 every time this app restarted, it would look for amount of already stored eds and start iterating from there until limit is reached.

I think it might be important, since we want to run this tool to find limits of storage and where things breaks. So it should be ready for failure state and do not expect to always stop gracefully. Would be nice to be able to resume it and restore previous run state.

Reply:
I like the idea. I will also then disable cleanup by default

Reply:
Maybe you might like an idea to replace "cleanup" option with "clean-start" instead, that will clear previous run data.

Reply:
Added `fresh` for similar effect